ign_rviz_plugins/include/ignition/rviz/plugins/PoseWithCovarianceDisplay.hpp

```diff
@@ -0,0 +1,298 @@
+
+  // Destructor
+  ~PoseWithCovarianceDisplay();
+
+  // Documentation Inherited
+  void LoadConfig(const tinyxml2::XMLElement * /*_pluginElem*/) override;
+
+  // Documentation Inherited
+  void initialize(rclcpp::Node::SharedPtr _node) override;
+
+  // Documentation Inherited
+  void callback(const geometry_msgs::msg::PoseWithCovarianceStamped::SharedPtr _msg) override;
+
+  // Documentation inherited
+  void setTopic(const std::string & topic_name) override;
+
+  // Documentation inherited
+  void subscribe() override;
+
+  // Documentation inherited
+  void reset() override;
+
+  /**
+   * @brief Set ROS Subscriber topic through GUI
+   * @param[in] topic_name ROS Topic Name
+   */
+  Q_INVOKABLE void setTopic(const QString & topic_name);
+
+  /**
+   * @brief Update subscription Quality of Service
+   * @param[in] _depth Queue size of keep last history policy
+   * @param[in] _history Index of history policy
+   * @param[in] _reliability Index of reliability policy
+   * @param[in] _durability Index of durability policy
+   */
+  Q_INVOKABLE void updateQoS(
+    const int & _depth, const int & _history, const int & _reliability,
+    const int & _durability);
+
+  /**
+   * @brief Qt eventFilters. Original documentation can be found
+   * <a href="https://doc.qt.io/qt-5/qobject.html#eventFilter">here</a>
+   */
+  bool eventFilter(QObject * _object, QEvent * _event);
+
+  // Documentation inherited
+  void setFrameManager(std::shared_ptr<common::FrameManager> _frameManager) override;
+
+  /**
+   * @brief Get the topic list as a string
+   * @return List of topics
+   */
+  Q_INVOKABLE QStringList getTopicList() const;
+
+  /**
+   * @brief Set pose visual shape
+   * @param _shape Visual shape. True: Arrow; False Axis
+   */
+  Q_INVOKABLE void setShape(const bool & _shape);
+
+  /**
+   * @brief Set axis arrow head visibility
+   */
+  Q_INVOKABLE void setAxisHeadVisibility(const bool & _visible);
+
+  /**
+   * @brief Set axis dimensions
+   * @param _length Axis length
+   * @param _radius Axis radius
+   */
+  Q_INVOKABLE void setAxisDimensions(const float & _length, const float & _radius);
+
+  /**
+   * @brief Set arrow dimensions
+   * @param _shaftLength Arrow shaft length
+   * @param _shaftRadius Arrow shaft radius
+   * @param _headLength Arrow head length
+   * @param _headRadius Arrow head radius
+   */
+  Q_INVOKABLE void setArrowDimensions(
+    const float & _shaftLength, const float & _shaftRadius,
+    const float & _headLength, const float & _headRadius);
+
+  /**
+   * @brief Set visual color and transparency
+   * @param _color Color and transparency of visual
+   */
+  Q_INVOKABLE void setColor(const QColor & _color);
+
+  /**
+   * @brief Set covariance visual visibility
+   */
+  Q_INVOKABLE inline void setCovVisible(const bool& visible) { std::lock_guard<std::mutex>(this->lock); covVisual->setCovVisible(visible); }
+  
+  /**
+   * @brief Set position covariance visual visibility
+   */
```

Comment:
```suggestion
  /**
   * @brief Set position covariance visual visibility
   * @param
   */
```

Also in the other methods and use `_`